Assets/_BForBoss/_Core/Scripts/UserInterface/GameplaySettingsViewBehaviour.cs

```diff
@@ -50,12 +50,10 @@ private void SetViews()
             _showRAMToggle.isOn = false;
         }
 
-        private void HideSettingsForProduction()
+        private void ShowSettingsForDevelopment()
```

Comment:
You can wrap this method with `#if DEVELOPMENT_BUILD || UNITY_EDITOR` since it will be useless in production anyway. 

Realistically the compiler will know that there is no reference to this and byte strip it away, but for the sake of consistency can just add the Pragma